app-services/cloud-event-transforms/res/docker/configuration.toml

```diff
@@ -0,0 +1,31 @@
+[Writable]
```

Comment:
Remove this file. We no longer create Docker versions. Instead we use env overrides in the docker compose files.